arcade/arcade/cli/utils.py

```diff
@@ -587,3 +596,73 @@ def delete_deprecated_config_file() -> None:
 
     if os.path.exists(deprecated_config_file_path):
         os.remove(deprecated_config_file_path)
+
+
+def get_user_input() -> str:
+    """
+    Get input from the user, handling multi-line input.
+    """
+    MULTI_LINE_PROMPT = '"""'
+    user_input = input()
+    # Handle multi-line input
+    if user_input.startswith(MULTI_LINE_PROMPT):
+        user_input = user_input[len(MULTI_LINE_PROMPT) :]
+
+        while not user_input.endswith(MULTI_LINE_PROMPT):
+            line = input()
+            if not line:
+                print()
+            user_input += "\n" + line
+
+        user_input = user_input.rstrip(MULTI_LINE_PROMPT)
+    else:
+        # Handle single-line input
+        while not user_input.strip():
+            user_input = input()
+
+    return user_input.strip()
+
+
+def display_chat_help() -> None:
+    """Display the help message for arcade chat."""
+    help_message = dedent(f"""\
+        [default]
+        Available Commands:
+          {ChatCommand.SHOW:<10} Show all available tools
+          {ChatCommand.CLEAR:<10} Clear the chat history
+          {ChatCommand.HELP:<10} Help for a command
+          {ChatCommand.EXIT:<10} Exit the chat
+
+        Use \"\"\" to begin & end a multi-line message[/default]
```

Comment:
Curious whether this is more standard, or shift-enter. I feel like I'm more used to doing shift-enter in apps where enter sends a message.

Either way, this PR is a great improvement. 👍

Reply:
I took inspiration from Ollama for the triple quotes.

I agree Shift+Enter is nicer, but I think that would require something more involved than using the builtin `input()`

Reply:
Makes sense 👍